float32 tx
float32 ty
float32 tz
float32 rx
float32 ry
float32 rz
int8 it_transl
int8 it_rot